Failure to resolve `lodash/flow` issue

Hi,

there appears to be an issue how `lodash/flow` is imported from some src files
```
Failed to resolve 'lodash/flow' from 
'./node_modules/react-keyed-file-browser/dist/react-keyed-file-browser.js'
```

To reproduce with parcel
1. In an empty directory create

`package.json`
``` json
{
  "devDependencies": {
    "parcel": "^2.12.0"
  },
  "dependencies": {
    "react": "17.0.2",
    "react-dom": "17.0.2",
    "react-keyed-file-browser": "^1.14.0"
  }
}
```

`src\index.js`
``` js
import ReactDOM from 'react-dom';

import FileBrowser from 'react-keyed-file-browser';

function App() {
  return <FileBrowser
    files={[]}
    />;
}

ReactDOM.render(
  <App />, document.getElementById('app')
);
```

`src/index.html`
``` html
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <title>My Parcel App</title>
  </head>
  <body>
    <div id="app"></div>
    <script type="module" src="index.js"></script>
  </body>
</html>
```
2.  Install packages
``` bash
$ yarn install
```
3. Bring the app up
``` bash
$ yarn parcel src/index.html 
yarn run v1.22.22
warning package.json: No license field
$ \issue-keyed-file-browser-lodashflow\node_modules\.bin\parcel src/index.html
Server running at http://localhost:1234
Building...
Ã— Build failed.

@parcel/core: Failed to resolve 'lodash/flow' from 
'./node_modules/react-keyed-file-browser/dist/react-keyed-file-browser.js'

  \issue-keyed-file-browser-lodashflow\node_modules\react-keyed-file-browser\dist\react-keyed-file-browser.js:1:1499
  > 1 | uire("lodash/flow")},function(e,t,r){"use strict";r.r(t),r.d(t,"RawFileBrowser",(function(){return Ft})),r.d(t,"Base
  >   |      ^^^^^^^^^^^^^

@parcel/resolver-default: Cannot find module 'lodash'
```

This is because the lodash dep in `react-keyed-file-browser` is `lodash.flow`, not the whole `lodash` package.

PR to follow.

--- a/src/files/list-thumbnail.js
+++ b/src/files/list-thumbnail.js
@@ -3,7 +3,7 @@ import ClassNames from 'classnames'
 import { DragSource, DropTarget } from 'react-dnd'
 import { NativeTypes } from 'react-dnd-html5-backend'
 import { formatDistanceToNow } from 'date-fns'
-import flow from 'lodash/flow'
+import flow from 'lodash.flow'
 
 import BaseFile, { BaseFileConnectors } from './../base-file.js'
 import { fileSize } from './utils.js'

--- a/src/files/table.js
+++ b/src/files/table.js
@@ -3,7 +3,7 @@ import ClassNames from 'classnames'
 import { DragSource, DropTarget } from 'react-dnd'
 import { NativeTypes } from 'react-dnd-html5-backend'
 import { formatDistanceToNow } from 'date-fns'
-import flow from 'lodash/flow'
+import flow from 'lodash.flow'
 
 import BaseFile, { BaseFileConnectors } from './../base-file.js'
 import { fileSize } from './utils.js'

--- a/src/folders/list-thumbnail.js
+++ b/src/folders/list-thumbnail.js
@@ -2,7 +2,7 @@ import React from 'react'
 import ClassNames from 'classnames'
 import { DragSource, DropTarget } from 'react-dnd'
 import { NativeTypes } from 'react-dnd-html5-backend'
-import flow from 'lodash/flow'
+import flow from 'lodash.flow'
 
 import BaseFolder, { BaseFolderConnectors } from './../base-folder.js'
 import { BaseFileConnectors } from './../base-file.js'

--- a/src/folders/table.js
+++ b/src/folders/table.js
@@ -2,7 +2,7 @@ import React from 'react'
 import ClassNames from 'classnames'
 import { DragSource, DropTarget } from 'react-dnd'
 import { NativeTypes } from 'react-dnd-html5-backend'
-import flow from 'lodash/flow'
+import flow from 'lodash.flow'
 
 import BaseFolder, { BaseFolderConnectors } from './../base-folder.js'
 import { BaseFileConnectors } from './../base-file.js'

